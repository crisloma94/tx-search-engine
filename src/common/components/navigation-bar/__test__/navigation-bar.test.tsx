import { render } from '@testing-library/react';
import { Provider } from 'react-redux';
import Theme from '../../../../styles/theme';
import { store } from '../../../../core/store/store';
import NavigationBar from '../navigation-bar.component';

describe('Navigation bar component test suite', () => {
  const getComponent = () => (
    <Provider store={store}>
      <Theme>
        <NavigationBar />
      </Theme>
    </Provider>
  );
  const setup = () => render(getComponent());

  it('Should match snapshot', () => {
    const { container } = setup();
    expect(container).toMatchSnapshot();
  });
});
