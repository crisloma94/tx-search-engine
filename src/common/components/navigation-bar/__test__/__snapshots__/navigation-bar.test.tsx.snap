// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navigation bar component test suite Should match snapshot 1`] = `
<div>
  <nav
    class="sc-kcuKUB gZBXOh"
  >
    <div
      class="sc-kPTPQs bmNzVv"
    >
      <img
        alt="Logo"
        src="test-file-stub"
      />
      <span>
        HappyBooks
      </span>
      <div
        class="sc-gTRrQi iSzjiZ"
      >
        <input
          class="sc-bgqQcB gWCqZe"
          name="search"
          placeholder="Search..."
          type="text"
          value=""
        />
      </div>
    </div>
  </nav>
</div>
`;
